tide_core.install

```diff
@@ -684,3 +685,45 @@ function tide_core_update_8026() {
   $config->set('display.default.display_options.empty.area.content.format', 'plain_text');
   $config->save();
 }
+
+/**
+ * Moves introduction up to top.
+ */
+function tide_core_update_8027() {
+  $node_types = NodeType::loadMultiple();
```

Comment:
@vincent-gao as this will load all the nodes and this update will run on all the nodes, so should we run it in batch size?

Reply:
not for all nodes, it is for node types. i think it doesn't need, very small load.

Reply:
My bad, I was reading it as `Node::loadMultiple()`. Looks fine to me.